packages/camera/camera/README.md

```diff
@@ -80,6 +80,28 @@ void didChangeAppLifecycleState(AppLifecycleState state) {
 }
 ```
 
+### Handling camera access permissions
+
+Permission errors may be thrown when initializing the camera controller, and you are expected to handle them properly.
+
+Here is a list of all permission error codes that can be thrown:
+
+- CameraAccessDenied
+Thrown when user denied the camera access permission.
```

Comment:
The newlines here aren't doing anything here in the output (see alternate view). You should instead format these as a single paragraph (which is what they render as), with at `:` after the code.